Examples/KlaviyoSwiftExamples/Shared/CheckOutViewController.swift

```diff
@@ -6,67 +6,70 @@
 //  Copyright © 2015 CocoaPods. All rights reserved.
 //
 
-import UIKit
 import KlaviyoSwift
+import UIKit
 
 class CheckOutViewController: UIViewController {
     // MARK: public members
+
     var cart: Cart!
-    
+
     // MARK: private members
-    private var numberOfItemsInCart: [MenuItem : Int]!
+
+    private var numberOfItemsInCart: [MenuItem: Int]!
     private var uniqueItemsArray: [MenuItem]!
     private var cartTotal: Double = 0
-    @IBOutlet private weak var orderTotalLabel: UILabel!
-    @IBOutlet private weak var tableview: UITableView!
-    
+    @IBOutlet private var orderTotalLabel: UILabel!
+    @IBOutlet private var tableview: UITableView!
+
     // MARK: view lifecycle
+
     override func viewDidLoad() {
         super.viewDidLoad()
         numberOfItemsInCart = cart.createUniqueDictionary
         uniqueItemsArray = cart.createCartItemsSet
-        
-        //Hide unused cells
+
+        // Hide unused cells
         let tblfooter = UIView(frame: CGRect.zero)
         tableview.tableFooterView = tblfooter
         tableview.tableFooterView?.isHidden = true
         tableview.backgroundColor = UIColor.clear
         tableview.reloadData()
     }
-    
+
     override func viewWillAppear(_ animated: Bool) {
         orderTotalLabel.text = "Order Total: \(cart.valueOfCart)"
     }
 
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
     }
-    
+
     // MARK: IBActions
-    
+
     /*
-    Check out button action
-    Triggers checkout completed event & empties the cart
-    */
+     Check out button action
+     Triggers checkout completed event & empties the cart
+     */
     @IBAction func checkOutButton(_ sender: UIButton) {
         if cart.cartItems.isEmpty {
             return
         }
-        
+
         // Empty the cartItems to 0 and save
         if !cart.cartItems.isEmpty {
             cart.cartItems.removeAll()
         }
         cart.saveCart()
 
         // Trigger "Checkout Completed" Event
-        //TODO: check with Noah why SDK doesn't take in swift dictionary
+        // TODO: check with Noah why SDK doesn't take in swift dictionary
         let propertiesDictionary = NSMutableDictionary()
         propertiesDictionary[Klaviyo.sharedInstance.KLEventTrackPurchasePlatform] = "iOS \(UIDevice.current.systemVersion)"
         propertiesDictionary["Total Price"] = cartTotal
-        propertiesDictionary["Items Purchased"] = uniqueItemsArray.compactMap { $0.name }
+        propertiesDictionary["Items Purchased"] = uniqueItemsArray.compactMap(\.name)
```

Comment:
@ndurell This lint change is causing a compiler error, may be because the example app isn't using the same version of swift?
<img width="1345" alt="image" src="https://user-images.githubusercontent.com/118314354/226929635-d3ca82e0-273f-49c6-93e7-e1b43ce0dc08.png">

Reply:
Essentially these two changes are causing the compiler on the example app to fail - 
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/118314354/226935298-2e491db6-b55c-4c23-afa0-559b37488344.png">

Are you seeing any issues on the SDK side?

Reply:
Nope...what version of swift are you using?